xml/System.Windows.Threading/Dispatcher.xml

```diff
@@ -1577,15 +1577,7 @@
         <typeparam name="TResult">The return value type of the specified delegate.</typeparam>
         <param name="callback">A delegate to invoke through the dispatcher.</param>
         <summary>Executes the specified <see cref="T:System.Func`1" /> synchronously on the thread the <see cref="T:System.Windows.Threading.Dispatcher" /> is associated with.</summary>
-        <returns>To be added.</returns>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- The value returned by `callback`.  
-  
- ]]></format>
-        </remarks>
+        <returns>The value returned by <paramref name="callback" />.</returns>
```

Comment:
@sambent Can you verify that this return value is correct? It looks like it was incorrectly placed in the Remarks before.